app/scripts/views/mixins/settings-mixin.js

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// helper functions for views with a profile image. Meant to be mixed into views.
+
+'use strict';
+
+define([
```

Comment:
Does this module have unit tests? It is fully covered by tests, I'm just wondering if the coverage is explicit.